sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -49,11 +51,16 @@ import {
   transformKeyValue,
   formatAcceptDateTime
 } from "./internal/helpers";
-import { tracingPolicy, isNode as isNodeFromCoreHttp } from "@azure/core-http";
+import { tracingPolicy, isNode as coreHttpIsNode } from "@azure/core-http";
 import { Spanner } from "./internal/tracingHelpers";
 import { GetKeyValuesResponse, AppConfigurationOptions as GeneratedAppConfigurationClientOptions } from "./generated/src/models";
 import { syncTokenPolicy, SyncTokens } from './internal/synctokenpolicy';
 
+const packageName = "azsdk-js-app-configuration";
+
+// this constant should always be the same as the package.json's version - we use it when forming the 
+// User - Agent header. There's a unit test that makes sure it always stays in sync.
+export const packageVersion = "1.0.0-preview.11";
```

Comment:
Will this show up in typedoc since it's exported?

Reply:
You know...it doesn't but just in case I'm going to give it the traditional internal/ignore treatment.